framework/test/unit/modules/token/commands/transfer.spec.ts

```diff
@@ -264,34 +257,6 @@ describe('Transfer command', () => {
 			);
 		});
 
-		it('should pass if token balance for token ID TOKEN_ID_STORE_INITIALIZATION is at least the sum of configured initialization fee and transaction amount', async () => {
```

Comment:
this test should still be valid

Reply:
It should use `TOKEN_ID_LSK` as token ID, instead of `TOKEN_ID_STORE_INITIALIZATION`, right?

Reply:
the test name should be
`should pass if token balance is at least the sum of configured initialization fee and transaction amount when feeTokenID and sending tokenID are the same`

Reply:
Updated